converter/dicom-cast/src/Microsoft.Health.DicomCast.TableStorage/Features/Storage/TableServiceClientProvider.cs

```diff
@@ -19,19 +22,29 @@ public class TableServiceClientProvider : IHostedService, IRequireInitialization
     {
         private readonly TableServiceClient _tableServiceClient;
         private readonly RetryableInitializationOperation _initializationOperation;
+        private readonly TableDataStoreConfiguration _tableDataStoreConfiguration;
+
+        public Dictionary<string, string> TableList { get; }
```

Comment:
Can you add a comment here to describe what the key and value should be? I think it is key = constant, value = prefix+constant but it's not clear